TheOracle2/Clock/CampaignClock.cs

```diff
@@ -30,6 +30,9 @@ public override ComponentBuilder MakeComponents()
         {
             menu.AddOption(IClock.ResetOption());
         }
-        return new ComponentBuilder().WithSelectMenu(menu);
+        return new ComponentBuilder()
+            .WithSelectMenu(menu)
+            // .WithButton(ILogWidget.ToggleAlertButton(AlertOnIncrement))
```

Comment:
sounds like we're going a different direction with log output, but if it's NBD i'd prefer to leave this here commented out for now.